workers_controller.js

```diff
@@ -96,40 +96,56 @@ SCWorker.create({
 				],
 			},
 			(err, scope) => {
+				scServer.addMiddleware(scServer.MIDDLEWARE_HANDSHAKE, (req, next) => {
```

Comment:
@diego-G Did an update of SocketCluster to `v10.0.1` recently, so you should rename `MIDDLEWARE_HANDSHAKE` to `MIDDLEWARE_HANDSHAKE_WS` (same thing, just new name).

Note that now there is also a SocketCluster-protocol-level handshake middleware (`MIDDLEWARE_HANDSHAKE_SC`) which behaves the same as `MIDDLEWARE_HANDSHAKE_WS` except that you can now send back errors.

That said, your solution looks pretty clean/logical so I think you can just leave it as is (just rename `MIDDLEWARE_HANDSHAKE` to `MIDDLEWARE_HANDSHAKE_WS`).

Reply:
The PR is not merged yet https://github.com/LiskHQ/lisk/pull/1633 but it will be soon if we're lucky.

Reply:
Waiting for the luck then

Reply:
It was merged.